synapse/rest/client/v2_alpha/auth.py

```diff
@@ -74,6 +75,14 @@ async def on_GET(self, request, stagetype):
             # re-authenticate with their SSO provider.
             html = await self.auth_handler.start_sso_ui_auth(request, session)
 
+        elif stagetype == LoginType.REGISTRATION_TOKEN:
+            html = self.registration_token_template.render(
+                session=session,
+                myurl="{}/r0/auth/{}/fallback/web".format(
```

Comment:
This hasn't got an unstable prefix or anything, but I don't think it can because clients use a standard form and just insert the auth type?